pkgs/os-specific/linux/decklink/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, lib, requireFile, fetchpatch, kernel }:
+
+stdenv.mkDerivation rec {
+  pname = "decklink";
+  major = "12.2";
+  version = "${major}a12";
+
+  src = requireFile {
+    name = "Blackmagic_Desktop_Video_Linux_${major}.tar.gz";
```

Comment:
```suggestion
    name = "Blackmagic_Desktop_Video_Linux_${major}.tar";
```

I get this error:

```
***
Unfortunately, we cannot download file Blackmagic_Desktop_Video_Linux_12.2.tar.gz automatically.
Please go to https://www.blackmagicdesign.com/support/download/33abc1034cd54cf99101f9acd2edd93d/Linux to download it yourself, and add it to the Nix store
using either
  nix-store --add-fixed sha256 Blackmagic_Desktop_Video_Linux_12.2.tar.gz
or
  nix-prefetch-url --type sha256 file:///path/to/Blackmagic_Desktop_Video_Linux_12.2.tar.gz

***
```

I thought the hash might not match, but it's actually the file name!

Even tho the direct download link includes `gz`, the file i have does not.

```
[davidak@gaming:~/Downloads]$ mv Blackmagic_Desktop_Video_Linux_12.2.tar Blackmagic_Desktop_Video_Linux_12.2.tar.gz

[davidak@gaming:~/Downloads]$ nix-store --add-fixed sha256 Blackmagic_Desktop_Video_Linux_12.2.tar.gz
warning: dumping very large path (> 256 MiB); this may run out of memory
/nix/store/vcwjf0h7jp3vgpvs0kd55m5mvr432bmp-Blackmagic_Desktop_Video_Linux_12.2.tar.gz
```

This actually solved the issue.



```
[davidak@gaming:~/Downloads]$ sha256sum Blackmagic_Desktop_Video_Linux_12.2.tar 
62954a18b60d9040aa4a959dff30ac9c260218ef78d6a63cbb243788f7abc05f  Blackmagic_Desktop_Video_Linux_12.2.tar
```